content/influxdb/v2.0/reference/upgrading/influxd-upgrade-guide.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Upgrade from InfluxDB 1.x to InfluxDB 2.0
+description: >
+  Upgrade from InfluxDB 1.x to InfluxDB 2.0.
+menu:
+  influxdb_2_0_ref:
+    parent: Upgrading InfluxDB
+    name: Upgrade from InfluxDB 1.x to InfluxDB 2.0
+weight: 9
+---
+
+Use the `influxd upgrade` command to upgrade an instance of InfluxDB 1.x to InfluxDB 2.0.
+This process will copy all data stored in [databases](/influxdb/v1.8/concepts/glossary/#database) and 
+[retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp) (used in 1.x)
+over to [buckets](/influxdb/v2.0/reference/glossary/#bucket) (used in 2.0).
+
+{{% warn %}}
+Be sure to backup all data before upgrading with `influx upgrade`.
+{{% /warn %}}
+
+1. Stop your running InfluxDB 1.x instance.
+2. If your configuration file is at the default location, run:
```

Comment:
include default location

Reply:
Hm. @russorat: just to be clear: is this referring to a 1.x or 2.0 configuration file?

(And where _is_ the default 2.0 configuration file? 🤔 )

Reply:
This might be different across distros/OS...